### ISSUE TITLE
Increase contrast of active/inactive buttons for accessibility

### ISSUE THREAD

**Issue by jdufresne** [fullcalendar/fullcalendar#5546]
Test case:

https://codepen.io/jdufresne/pen/ZEQymdO?editable=true&editors=001%3Dhttps%3A%2F%2Ffullcalendar.io%2Freduced-test-cases

Actual:

![cal-buttons](https://user-images.githubusercontent.com/347634/85773315-8f319480-b6d2-11ea-8854-6ea8dc3ea7a6.png)

The contrast between the active button and non-active button is very low. For users with a11y needs, it is difficult to notice the distinction between these elements.

I recommend the default contrast be increased.

**Comment by acerix** [comment]
I agree the default theme does not have much contrast there, although it's better in the Bootstrap theme and the colours can also be changed with CSS. 

https://fullcalendar.io/docs/bootstrap-theme-demo

https://codepen.io/acerix/pen/rNxwoVy?editors=0110
